web/satellite/src/main.ts

```diff
@@ -15,8 +16,10 @@ Vue.config.performance = true;
 Vue.config.productionTip = false;
 
 const notificator = new NotificatorPlugin();
+const segmentio = new SegmentioPlugin();
```

Comment:
mb we can name it segment?

Reply:
the reason for naming it segmentio is because that's what this plugin called. I don't want it to be confused with the concept of a segment in our network